release notes/v0.49.0.md

```diff
@@ -89,9 +89,10 @@ The migration is straightforward. Just replace `k6/experimental/grpc` with `k6/n
 
 ### Collect usage data on imported internal modules and outputs [#3525](https://github.com/grafana/k6/pull/3525)
 
-// TODO fill with info
-
-
+Added the data collection of the used modules and outputs when the [usage report](https://grafana.com/docs/k6/latest/misc/usage-collection) is enabled. The data collection is only related to the built-in k6 modules and outputs. Private, custom modules and extensions [are never collected](https://github.com/grafana/k6/blob/f35e67902605877ebf2c5e9c8673cd7faf4cdc1e/cmd/report.go#L33-L57). 
+The usage report is enabled by default in k6, but it is possible to opt-out using the [no-usage-report](https://grafana.com/docs/k6/latest/using-k6/k6-options/reference/#no-usage-report) option.
+We always want to improve the product, but at the same time we need to pay attention where we allocate our resources. Having data of what are the most used modules and outputs give us a better confidence to make decisions because we are supported by data.
+It will let us know what is the percentage of our users that will benefit from the introduction of a new feature and on the other side, in case of a breaking change, how many of them would be impacted.
```

Comment:
If you meant for this to be separate paragraphs - you need a line in between them.

Currently it doesn't look very nice 
![Uploading image.png…]()

Reply:
Yes, that was the intention. Added the new line.